test/component/testThankyouPage.js

```diff
@@ -15,19 +15,22 @@ describe('thank-you', () => {
     });
 
     describe('Verify Content, Errors and Redirection', () => {
-
         it('test right content loaded on the page when NO soft stop', (done) => {
             const sessionData = {
                 applicant: {
                     nameAsOnTheWill: 'Yes'
                 },
+                ccdCase: {
+                    id: '1234-5678-9012-3456'
+                }
             };
             testWrapper.agent.post('/prepare-session/form')
                 .send(sessionData)
                 .end(() => {
                     const excludeKeys = ['stopParagraph1'];
```

Comment:
As `stopParagraph1` has been removed from the content, we can remove it from being excluded in this test